default_warehouse_from_sale_team/__openerp__.py

```diff
@@ -15,6 +15,7 @@
         'sales_team',
         'base_action_rule',
         'purchase_requisition',
+        'warning',
```

Comment:
@JulioSerna I have to add this depend [odoo/addons/warning](https://github.com/odoo/odoo/tree/9.0/addons/warning) because in test when is created the new user this error appears:

<img width="1278" alt="skitch_13" src="https://cloud.githubusercontent.com/assets/11341058/18555830/466f3ad4-7b37-11e6-9555-300ed2617a0d.png">